src/Symfony/Component/Translation/Extractor/PhpExtractor.php

```diff
@@ -262,7 +262,7 @@ private function getValue(\Iterator $tokenIterator): string
     /**
      * Extracts trans message from PHP tokens.
      */
-    protected function parseTokens(array $tokens, MessageCatalogue $catalog, string $filename)
+    protected function parseTokens(array $tokens, MessageCatalogue $catalog, string $filename): void
```

Comment:
BC break?

Reply:
Good spot, should be PHPdoc.

(guess I was put on a wrong foot by all the private methods in the file before this one)